hack/extract_images.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# The script reports:
+# 1. Images used by the Kubeflow Working Groups
+# 2. All images used by Kubeflow
+# The reported image lists are saved in respective files under ../doc directory
+#The script must be executed from the `hack` folder as it use relative paths
+
+# Future release process enhancements may include an automatic image inventory scan, generating SBOM files
+# vulnerability scanning and managing licenses.
+
+version=latest
+images=()
+
+declare -A wg_dirs=(
+  [automl]="../apps/katib"
+  [pipelines]="../apps/pipeline/upstream ../apps/kfp-tekton/upstream"
+  [training]="../apps/training-operator/upstream"
+  [manifests]="../common ../example"
+  [workbenches]="../apps/admission-webhook/upstream ../apps/centraldashboard/upstream ../apps/jupyter/jupyter-web-app/upstream ../apps/volumes-web-app/upstream ../apps/tensorboard/tensorboards-web-app/upstream ../apps/profiles/upstream ../apps/jupyter/notebook-controller/upstream ../apps/tensorboard/tensorboard-controller/upstream"
+  [serving]="../contrib/kserve"
```

Comment:
since KServe is part of `/contrib`, does it still make sense to include it here?

Reply:
Thanks for pointing that out, we can leave it in then. it would be easy to take it out in the future if decided to remove it.